src/lib/UI/Dataset/VersionsDataset.php

```diff
@@ -41,9 +41,15 @@ public function __construct(ContentService $contentService, ValueFactory $valueF
      */
     public function load(ContentInfo $contentInfo): self
     {
+        $versions = $this->contentService->loadVersions($contentInfo);
+
+        if ($versions instanceof \Traversable) {
+            $versions = iterator_to_array($versions);
+        }
```

Comment:
We have a dedicated polyfill, so the following should be enough (ref: https://github.com/ibexa/polyfill-php82/pull/1):
```suggestion
        $versions = iterator_to_array($versions);
```